dev-util/sidreloc/sidreloc-1.0.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="Tool for relocating SID tunes"
+DESCRIPTION="Sidreloc is a tool for relocating SID tunes any number of whole pages, \
+using a novel but remarkably simple algorithm. It can also relocate \
+all zero-page variables used by the tune."
+HOMEPAGE="https://www.linusakesson.net/software/sidreloc/index.php"
+COPYRIGHT="2012-2020 Linus Akesson
+	2005-2020 Ian Piumarta"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://hd0.linusakesson.net/files/sidreloc-$portVersion.tgz"
+CHECKSUM_SHA256="8ca55fb4886bda2a499f837e2f9ffd0a4b7217ee7bb1907ceed9e87ef6157bf6"
+
+PATCHES="sidreloc-$portVersion.patchset"
+ADDITIONAL_FILES="
+	configure.ac
+	Makefile.am
+	"
+
+ARCHITECTURES="!x86_gcc2 x86_64 ?arm ?ppc ?sparc"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	sidreloc$secondaryArchSuffix = $portVersion
+	cmd:sidreloc$commandSuffix = $portVersion
```

Comment:
this should be cmd:sidreloc since you put it in bin even when building ofr secondary architecture

Reply:
I'm not sure I undestood what you mean. I'm copying into $commandBinDir. Perhaps I did it in the wrong way?

`runConfigure --omit-dirs binDir ./configure --bindir="$commandBinDir"`

Reply:
Well $commandSuffix will be empty in all cases. But I see that other recipes do it the same way, so...